ext/search_kit/Civi/Api4/Action/SearchDisplay/GetSearchTasks.php

```diff
@@ -71,6 +71,13 @@ public function _run(\Civi\Api4\Generic\Result $result) {
       'number' => '>= 0',
     ];
 
+    $tasks[$entity['name']]['process'] = [
```

Comment:
This task only exists for the Afform entity right? You've added this for all entities.

Reply:
Yes, this exists only for `AfformSubmission` entity.

Reply:
Ok at the very least it should move into the below section for Entity-Specific actions, but per the code comment there, it's better to add them via hooks to prevent the coupling.

Reply:
I have removed this code and now adding it via hook.